src/embezzler.ts

```diff
@@ -169,6 +180,127 @@ export const embezzlerSources = [
     [],
     true
   ),
+  new EmbezzlerFight(
+    "Time-Spinner",
+    () =>
+      have($item`Time-Spinner`) &&
+      cliExecuteOutput("timespinner list monsters").includes(
+        $monster`Knob Goblin Embezzler`.name
+      ) &&
+      get("_timeSpinnerMinutesUsed") <= 7,
+    () =>
+      have($item`Time-Spinner`) &&
+      cliExecuteOutput("timespinner list monsters").includes($monster`Knob Goblin Embezzler`.name)
+        ? Math.min((10 - get("_timeSpinnerMinutesUsed")) / 3)
```

Comment:
Two stupid questions:
1. In line 188, we split the arg of includes onto its own line, but we don't do that in 193. Is that the intended style? I didn't see a linter around here, but I didn't look for one either.
2. What are we doing with time spinner minutes? I understood that the first one is to check that we have enough minutes to do this. Is this to tell us how many times we are going to timespin the Embez?

Reply:
1. It's what the linter wants. Very weird choice, though! I assume it's the difference between the ternary vs continuing the condition
2. If you view it in VSCode you can see the documentation Manendra lovingly crafted for the EmbezzlerFight class; the first arrow function is whether the fight is available right now, the second tells us the number of fights we expect to be able to get from it.

Reply:
1. OK.
2. Oh that's what is happening here. I thought we were  doing some pattern matxhing thing. The syntax reminds me of Haskel. So these are 0 argument lambdas that give us what we want. I assume this is the pattern that is generally used around here etc. Do we know that things like "cliExecuteOutput(timespinner list monsters)" is fast? Bc it looks like we are doing it twice and could theoretically cache the value. I haven't done any work on profiling this sort of thing, so it might turn out that this is actually more time efficient if storing takes time for some reason etc.

Reply:
I have no idea if it's fast, and in fact it doesn't even work right now! Would love an alternative.
I'm concerned about caching because garbo is meant to be re-entrant and this section is trying to be as stateless as it can be. I want it to check every iteration whether it currently is able to spin an Embezzler.